utils/avd_stop.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+if [[ -z "${ANDROID_HOME}" ]]; then
+    SDKDIR=$PWD/.android-sdk
+    export ANDROID_HOME=${SDKDIR}
+    export ANDROID_SDK_ROOT=${SDKDIR}
+fi
+
+echo "Killing previous emulators"
+${ANDROID_HOME}/platform-tools/adb devices | grep emulator | cut -f1 | while read line; do ${ANDROID_HOME}/platform-tools/adb -s $line emu kill; done
```

Comment:
Why is this a separate file? Looks only used on one place and makes it hard to follow `utils/avd_start.sh`?

Reply:
It's helpful when running avd locally.